src/api/github/org.ts

```diff
@@ -8,12 +8,23 @@ import {
   GitHubOrgConfig,
 } from '@/types';
 
+import { FORCE_USER_TOKEN_GITHUB_CLIENT } from '../../config';
```

Comment:
```suggestion
import { FORCE_USER_TOKEN_GITHUB_CLIENT } from '@/config';
```

Reply:
Done in 836c0050c20a4d9d9a3d800799df4697aa068ef8.